assignments/ShapeIndexing/scriptClassification.sh

```diff
@@ -11,7 +11,7 @@ NBIMGTESTS=40
 NBTESTS=5
 
 #No noise here
-MAXNOISE=0.0
+MAXNOISE=0.5
```

Comment:
well... let's assume that there is no noise in the classification test (remember, half of the image database is hidden :smile:). I'll edit your file.
Thanks for the getRank edit.